uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/bootstrap.php

```diff
@@ -9,8 +9,17 @@
 require_once DRUPAL_ROOT . '/includes/bootstrap.inc';
 drupal_bootstrap(DRUPAL_BOOTSTRAP_FULL);
 
+addCiviHRExtensionToIncludePath('/uk.co.compucorp.civicrm.hrcore/');
```

Comment:
Is this right? I thought the extension is supposed to be installed if needed rather than including its files directly?

Reply:
Yeah, this was removed since then